pulsar-broker/src/test/java/org/apache/pulsar/broker/auth/MockedPulsarServiceBaseTest.java

```diff
@@ -708,5 +708,14 @@ public static class ServiceProducer {
         private PersistentTopic persistentTopic;
     }
 
+    protected void sleepSeconds(int seconds){
+        try {
+            Thread.currentThread().sleep(1000 * seconds);
```

Comment:
Why need to use `Thread.currentThread().sleep(1000 * seconds);` instead of `Thread.sleep(1000 * seconds);`?

Reply:
Sure, instead